### ISSUE TITLE
Results of Fine-tuned model changes in every run

### ISSUE THREAD

**Issue by cagrikymk** [huggingface/pytorch-pretrained-BERT#565]
After I load the model with:
`
model = BertForNextSentencePrediction.from_pretrained("bert-base-uncased",state_dict=model_state_dict)
model.eval()
`
The prediction results are not stable. They change dractically in every run.

It gets stable if I fix the seed but I dont understand why we need that. Isnt the model supposed to be fixed since we are just evaluating?
